### PR TITLE
Schedule google.cloud for removal from Ansible 12

### PR DESCRIPTION
**Pull request by gotmax23**
Relates: https://forum.ansible.com/t/8609
Relates: https://github.com/ansible-community/ansible-build-data/issues/476

### CODE REVIEW

**Review by felixfontein — APPROVED** [review]
Should we wait with merging this until https://forum.ansible.com/t/9684/10 happened? Assuming that meeting happens before the next Ansible releases (that aren't pre-releases).

**Review by mariolenz — APPROVED** [review]
> Should we wait with merging this until https://forum.ansible.com/t/9684/10 happened? Assuming that meeting happens before the next Ansible releases (that aren't pre-releases).

The vote about  removing the collection has been accepted, so I think we should do it. If we make a decision, I think we should follow it!

**Comment by gotmax23** [comment]
> The vote about removing the collection has been accepted, so I think we should do it. If we make a decision, I think we should follow it!

Yeah, I agree, unless there's some strong reason not to, in which case that should be discussed in a community topic.

/cc @gundalow

**Comment by felixfontein** [comment]
In that case, feel free to merge this.

File: 11/collection-meta.yaml [hunk 1]
```diff
@@ -209,6 +209,14 @@ collections:
     repository: https://github.com/fortinet-ansible-dev/ansible-galaxy-fortios-collection
   google.cloud:
     repository: https://github.com/ansible-collections/google.cloud
+    removal:
+      major_version: 12
+      announce_version: 11.0.0a2
```
Comment (felixfontein) [inline 11/collection-meta.yaml:214]: Assuming there is no further alpha release, this needs to be updated to:
```suggestion
      announce_version: 11.0.0b1
```

**Comment by felixfontein** [comment]
@gotmax23 @mariolenz thanks!

**Pull request merged**